cfme/tests/intelligence/reports/test_import_export_reports_widgets.py

```diff
@@ -4,14 +4,26 @@
 from cfme import test_requirements
 from cfme.intelligence.reports.reports import ImportExportCustomReportsView
 from cfme.intelligence.reports.widgets import ImportExportWidgetsCommitView
+from cfme.utils.conf import cfme_data
+from cfme.utils.ftp import FTPClientWrapper
+from cfme.utils.ftp import FTPException
+from cfme.utils.log import logger
 from cfme.utils.path import data_path
 
 pytestmark = [pytest.mark.tier(1), test_requirements.report]
 
 
 @pytest.fixture()
 def widget_file(appliance):
-    file_path = data_path.join("ui/intelligence/import_widget.yaml").realpath().strpath
+
+    try:
+        fs = FTPClientWrapper(cfme_data.ftpserver.entities.reports)
+        file_path = fs.download("import_widget.yaml", "/tmp/import_widget.yaml")
+    except (FTPException, AttributeError) as e:
+        logger.warning("'import_widget.yaml' download failed with FTP: %s", e)
```

Comment:
Please use logger.exception here so we get better exception context/traceback automatically.

Reply:
Pushed a commit for this, and moved the file names to variables since they're repeated.